src/main/java/com/cgdp/library/converters/AuthorDtoConverter.java

```diff
@@ -0,0 +1,22 @@
+package com.cgdp.library.converters;
+
+import com.cgdp.library.dto.CreateAuthorRequestDTO;
+import com.cgdp.library.entity.AuthorEntity;
+import lombok.AllArgsConstructor;
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Component;
+
+@AllArgsConstructor
+@Component
+public class AuthorDtoConverter {
+
+    private final ModelMapper modelMapper;
+
+    public CreateAuthorRequestDTO toDTO(AuthorEntity authorEntity) {
```

Comment:
This conversion will never happen, the server itself will never have to create the request object